### DISCUSSION TITLE
第一次使用的一些问题

### DISCUSSION THREAD

**Discussion by JerryWuZiJie** [asdf2014/algorithm#145] [Q&A] [answered]
想问一下第一次用，已经被邀请成为collaborator了以后是要自己在Codes文件夹下建立一个自己名字的目录吗？
另外每天做题提交有什么具体要求吗？

**Comment by asdf2014** [comment, marked as answer]
@JerryWuZiJie 欢迎加入 :clap::clap::clap:
1. 是的，详见：https://github.com/asdf2014/algorithm/blob/master/README.md#way-to-participate
2. 有的，详见：https://github.com/asdf2014/algorithm/discussions/89
